tests/cases/standalone/common/basic.result

```diff
@@ -4,7 +4,7 @@ CREATE TABLE system_metrics (
     cpu_util DOUBLE,
     memory_util DOUBLE,
     disk_util DOUBLE,
-    ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP(),
```

Comment:
Is `CURRENT_TIMESTAMP` an invalid syntax now? Maybe we need to update the document if it is an expected behavior.

Reply:
y